pkg/cli/subcommands/taggerlist/command.go

```diff
@@ -76,19 +76,26 @@ func taggerList(_ log.Component, config config.Component, _ *cliParams) error {
 		return err
 	}
 
-	ipcAddress, err := pkgconfigsetup.GetIPCAddress(pkgconfigsetup.Datadog())
+	url, err := getTaggerURL(config)
 	if err != nil {
 		return err
 	}
 
+	return api.GetTaggerList(color.Output, url)
+}
+
+func getTaggerURL(_ config.Component) (string, error) {
+	ipcAddress, err := pkgconfigsetup.GetIPCAddress(pkgconfigsetup.Datadog())
```

Comment:
This should be using the `config.Component` being passed in, instead of retrieving the global object.

```suggestion
func getTaggerURL(cfg config.Component) (string, error) {
	ipcAddress, err := pkgconfigsetup.GetIPCAddress(cfg)
```

Reply:
You're absolutely right ! What a shame to not use it whereas it's already provided 😅
Done it in 1c1d46ef2849c956c29bbcad9c170be1870b1c47.